docs/cumulusci_flow.rst

```diff
@@ -36,36 +36,56 @@ When a Pull Request is approved and passing build, it is merged using the Merge
+
+CumulusCI facilitates working with feature branches (mainly) through two default flows:
 
 * **dev_org**: Used to deploy the unmanaged code and all dependencies from the feature branch into a Salesforce org to create a usable development environment.
 * **ci_feature**: Deploys the unmanaged code and all dependencies into a Salesforce org (typically a fresh scratch org) and run the Apex tests.  This flow is typically run by your CI app on new commits to any feature branch.
 
-Auto-Merging Main to Feature Branches
-=======================================
+Auto Merging
+============
+CumulusCI helps to keep large diffs and merge conflicts from being the norm. CumulusCI's auto-merge functionality helps teams:
+
+   * Keep feature branches up-to-date with the ``main`` branch (main to feature merges)
+   * Manage long-lived feature branches for larger features worked on by multiple developers (parent to child merges)
+   * Mange large releases that occur several times a year (release to future release merges).  
 
-One of the bigger differences between CumulusCI Flow and Github Flow or git-flow is that we automate the merging of main commits into all open feature branches the initial main build passes.  This auto-merge does a lot for us:
 
-* Ensures feature branches always track with main
+Main to Feature Merges 
+----------------------
+
+One of the bigger differences between CumulusCI Flow and Github Flow or git-flow is that we automate the merging of commits to a projects main branch into all open feature branches.  This auto-merge does a lot for us:
+
+* Ensures feature branches are in sync with the  main branch.
 * Re-tests each feature branch with any changes to main since the merge generates a new commit
 * Eliminates merge conflicts when merging a Pull Request to main
 
 To understand the benefit of auto-merging to feature branches, consider the following scenario: A developer starts work on a feature branch, puts in a few weeks on it, and then has to leave unexpectedly for a few months.  While they are on leave, their feature branch gets automatically updated with any new commits on main and rebuilt.  A few weeks into their leave, a new commit on main gets merged to their feature branch and breaks the build.  When the developer returns after their leave, they can look at the build history to find which commit from main broke their feature branch.
 
 Without auto-merging, the developer would return, merge main into their feature branch, and then have to sift through all the commits to main during their leave to figure out which one broke their feature branch.  More testing and build history is always a good thing in addition to the other benefits we gain from auto-merging.
 
-CumulusCI and Auto-Merging to Feature Branches
-----------------------------------------------
+CumulusCI facilitates the auto-merge to feature branches via the ``github_master_to_feature`` task which is included by default in the ``release_beta`` flow.
```

Comment:
Should we rename this task to github_automerge_main?

Reply:
I was thinking `github_automerge_maint_to_feature` and `github_automerge_parent_to_children`.